classes/app.rb

```diff
@@ -1,18 +1,28 @@
 require_relative './item'
```

Comment:
Hi @brytebee, 

Well done :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Dejan Vujovic_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Reply:
Hello, @VuDej👋👋,

Thanks for the warm-hearted review 👀👀.

![](https://media.giphy.com/media/JlpjgShzDsrMIyFu5U/giphy.gif)

I'll go ahead to merge and move on to the next milestone 🏁🏁.

**_Happy Hacking 😄😄_**